be/src/fs/hdfs/hdfs_fs_cache.cpp

```diff
@@ -82,6 +82,11 @@ static Status create_hdfs_fs_handle(const std::string& namenode, const std::shar
                               hedged_read_threshold_millis.data());
     }
 
+    hdfsBuilderConfSetStr(hdfs_builder, "fs.hdfs.impl", "com.starrocks.fs.StarRocksFileSystem");
```

Comment:
You can use `hdfsBuilderSetForceNewInstance(hdfs_builder)`, so here will always create new FileSystem, you don't need to use `fs.xxxx.impl.disable.cache`